hpx/parallel/algorithms/find.hpp

```diff
@@ -49,11 +49,15 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v1)
             parallel(ExPolicy const& policy, InIter first, InIter last,
                 T const& val)
             {
+                typedef detail :: algorithm_result<ExPolicy, InIter> result;
                 typedef typename std::iterator_traits<InIter>::value_type type;
```

Comment:
Could you please stick to the code formatting we use everywhere else (i.e. no spaces surrounding `::`)?